### ISSUE TITLE
Warzone Subdivision 

### ISSUE THREAD

**Issue by Alramore** [SmashMC-Development/Suggestions#1002]
This is more toward WZ but its a side of warzone, My suggestion is for some people going to warzone its not about the reputation but more for The legends, the extra Pokémon! A lot of people like my Self and my buddy Zoddity, we enjoy seeing the Pokémon and catching new Pokémon you don't see as often in wild. I think it should be called the Safari Zone, Somewhere people don't have to worry about there Pokémon being taken or Reputation, A place where we can fight over legends, and rarer Pokémon. 

Cost: 6k for 24hrs or until you log out
Pokémon lvls: definitely should be lower being that your not battling each other normal levels for legendary (70) and random for regular Pokémon

This should Apply to all Pokémon servers for everyone to enjoy

Additional Content
The Staff can throw in any other cool add-ons and such but i think this is a really cool idea for people that don't want the competitive side of pixelmon

**Comment by twizzy0** [comment]
no just no.. that would be boring as hell.

**Comment by Alramore** [comment]
but you dont think its boring when you constantly get pokemon taken from you? or legendaries? especially for people that arnt into the Ivs and EVs area of pixelmon?

**Comment by Alramore** [comment]
this isnt an action packed game, people dont care to battle, you want action go play an fps game, people just wanna catch pokemon and not have to deal with pokemon being stolen

**Comment by Alramore** [comment]
not to mention i didnt say take warzone out, but add this for the people who dont wanna battle but catch pokemon peacfully

**Comment by oLimooo** [comment]
I disagree with this suggestion. I don’t believe there should be a area with the same rewards as war zone without having any risk. War zone is meant to have risk because the chances of getting rare pokemon become commoner than in the wild. Having this “safari” will out shine war zone and will allow players to easily obtain legendaries. Smash is meant to be semi grindly, and I believe this area is just to easy, ruining the experience. 

If you want a non competitive side, check out /poketop, it allows you to gain reward without having to go into war zone, however can be beneficial to go into war zone to increase your chances at winning.

**Comment by ajdawg48** [comment]
Safari used to be a thing it had all biomes, pokemon around level 20-50, and was a pretty big place. It didn’t spawn legends so it wasn’t very op. So a place like that making a return might be a decent idea, but another version of warzone with just no risk and spawning legends would just defeat the whole point of wz. So would have to disagree with adding something like that also.

**Comment by XITACE** [comment]
Ur posting this because you are bad. One thing, GET GOOD

**Comment by WalkAway7** [comment]
> 

Yeah completely agree with what limo said here.

**Comment by Mikirae** [comment]
> Ur posting this because you are bad. One thing, GET GOOD

Don't act like this on the forums or I'm going to yoink your talking privileges

**Comment by Mikirae** [comment]
We can look in to readding the Safari Zone, but it will be legend-free :)
